client/layout/guided-tours/tours/simple-payments-tour.js

```diff
@@ -0,0 +1,61 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { translate } from 'i18n-calypso';
+import {
+	overEvery as and,
+} from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	makeTour,
+	Tour,
+	Step,
+	ButtonRow,
+	Link,
+	Quit,
+} from 'layout/guided-tours/config-elements';
+import {
+	isNotNewUser,
+} from 'state/ui/guided-tours/contexts';
+import { isDesktop } from 'lib/viewport';
+
+export const SimplePaymentsTour = makeTour(
+	<Tour
+		name="simplePaymentsTour"
+		version="20170816"
+		path="/post/"
+		when={ and( isDesktop, isNotNewUser ) }
+	>
+		<Step
+			name="init"
+			arrow="top-left"
+			target=".editor-html-toolbar__button-insert-media, .mce-wpcom-insert-menu button"
+			placement="below"
+			style={ { marginLeft: '22px', zIndex: 'auto' } }
+		>
+			<p>
+				{
+					translate( 'Did you know? ' +
```

Comment:
Similar suggestion to the one left on 17214 - We might be able to one-line the copy by nudging it down from `translate(` like so:
```js
{
  translate( 
    'Did you know? If your site is on the Premium or Business plan, you can add {{strong}}payment buttons{{/strong}} here!',
    {
      components: { ... }
    }
  )
}
```
Not blocking btw